data/human/engines.txt

```diff
@@ -1052,6 +1052,104 @@ outfit "A865 Atomic Steering"
 	description "Atomic engines are considered by many to be Deep Sky's crowning achievement. Although they are so expensive that relatively few of them are in use, there are few captains who do not dream of one day installing a set of them."
 
 
+# Note on Lateral Thrusters: My initial values here are to have each set of lateral thrusters be based on the thruster of the same "rank".
+# So the tiny lateral has values equivalent to about 90% of a tiny thruster. The small lateral thruster is roughly 80% of a small thruster, etc.
+# This continues by 10% increments down to the Huge lateral thrusters that are equivalent of 50% of the huge thrusters.
+# This percentage is applied equally to mass, outfit space, engine capacity, lateral thrust, lateral thrusting energy, and lateral thrusting heat.
+
+
+
+outfit "AL190 Atomic Lateral Thruster"
```

Comment:
Why 190 instead of 159?

Reply:
I was initially going to go with 190, 290, 390, etc. But then realized that the atomics for some reason just seem to be designed to be confusing with their 120, 250, 370, etc

So I went with using the AL prefix (following the example of the Reverse), and just replace the tail 0 with a 9, following the example of the steering that uses 5 in the same way.

So yeah, this 190 was just the first idea that got missed on revision. Latest push has it fixed. (as well as the sales list fix, too.)